integration/raft/channel_participation_test.go

```diff
@@ -93,105 +101,78 @@ var _ = Describe("ChannelParticipation", func() {
 		})
 
 		It("joins application channels using the channel participation API from genesis block", func() {
-			orderer := network.Orderer("orderer")
+			orderer1 := network.Orderer("orderer1")
+			orderer2 := network.Orderer("orderer2")
+			orderer3 := network.Orderer("orderer3")
+			orderers := []*nwo.Orderer{orderer1, orderer2, orderer3}
 			peer := network.Peer("Org1", "peer0")
+			for _, o := range orderers {
+				startOrderer(o)
+				nwo.ChannelParticipationList(network, o, nil)
+			}
 
-			By("joining channel as a member")
-			genesisBlock := applicationChannelGenesisBlock(network, orderer, peer, "participation-trophy")
+			genesisBlock := applicationChannelGenesisBlock(network, orderers, peer, "participation-trophy")
 			expectedChannelInfo := nwo.ChannelInfo{
 				Name:            "participation-trophy",
 				URL:             "/participation/v1/channels/participation-trophy",
 				Status:          "active",
 				ClusterRelation: "member",
 				Height:          1,
 			}
-			nwo.ChannelParticipationJoin(network, orderer, "participation-trophy", genesisBlock, expectedChannelInfo)
-			nwo.ChannelParticipationListOne(network, orderer, expectedChannelInfo)
+
+			for _, o := range orderers {
+				By("joining " + o.Name + " to channel as a member")
+				nwo.ChannelParticipationJoin(network, o, "participation-trophy", genesisBlock, expectedChannelInfo)
+				nwo.ChannelParticipationListOne(network, o, expectedChannelInfo)
+			}
 
 			By("waiting for the leader to be ready")
-			findLeader([]*ginkgomon.Runner{ordererRunner})
+			findLeader(ordererRunners)
 
-			By("ensuring the channel is usable by submitting a transaction")
+			By("ensuring the channel is usable by submitting a transaction to each orderer")
 			env := CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello"))
-			resp, err := nwo.Broadcast(network, orderer, env)
-			Expect(err).NotTo(HaveOccurred())
-			Expect(resp.Status).To(Equal(common.Status_SUCCESS))
-			waitForBlockReception(orderer, peer, network, "participation-trophy", 1)
+			expectedBlockNum := 1
+			for _, o := range orderers {
+				resp, err := nwo.Broadcast(network, o, env)
```

Comment:
And again.

Please stop extending nwo.

Reply:
This was introduced before this PR but I've moved this Broadcast and Deliver function to its own package.